packages/itwinui-css/src/input/input.scss

```diff
@@ -27,7 +27,8 @@
   @include iui-input-size;
 
   &::placeholder {
-    --_iui-input-text-color: var(--iui-color-text-disabled);
+    --_iui-input-text-color: var(--iui-color-text-placeholder);
+    font-weight: var(--iui-font-weight-light);
```

Comment:
lets also set `opacity: 1` to make it consistent in firefox. see https://github.com/iTwin/iTwinUI/discussions/1417

Reply:
Done!